pages/admin/index.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+import ApplicationsList from 'components/ApplicationsList/ApplicationsList';
+
+const AdminPage = ({ name }) => {
+  const [applications, setApplications] = useState();
+  useEffect(() => {
+    async function fetchData() {
+      const { data } = await axios.get('/api/applications');
```

Comment:
Should we be checking the response code here and showing a helpful error if the `statusCode` is not 200?

Reply:
This should be wrapped in a try/catch, but it's just a demo page.

Reply:
OK. I will approve on the basis that we are under the hammer and there can be error handling the next time this page is worked on.